scripts/badger/process_bribes_graviaura.py

```diff
@@ -58,6 +63,17 @@ def claim_and_sell_for_weth():
     SAFE.post_safe_tx()
 
 
+def ragequit(token_list=r.bribe_tokens_claimable_graviaura.values()):
+    DEV.take_snapshot(token_list)
+    for token in token_list:
```

Comment:
This assumes that we will always be claiming these bribe tokens. I would instead check if there's a balance of these on the processor and only rage quit on a token if that's the case.

Reply:
Handled [here](https://github.com/Badger-Finance/badger-avatars/pull/9)

Reply:
yea the idea was to use the `token_list` arg in a case like that, but a programmatic check would indeed make things a bit easier :) solved in f73f7e1